core/templates/pages/signup-page/signup-page.component.ts

```diff
@@ -30,6 +30,8 @@ import { LicenseExplanationModalComponent } from './modals/license-explanation-m
 import { RegistrationSessionExpiredModalComponent } from './modals/registration-session-expired-modal.component';
 import { SignupPageBackendApiService } from './services/signup-page-backend-api.service';
 
+declare const CAN_SEND_ANALYTICS_EVENTS: boolean;
```

Comment:
Ditto. Should use AnalyticsConstants and import the usual way? Let's not add new patterns or do things globally.